services/Directory/data/FilterList.json

```
@@ -17341,5 +17341,22 @@
     "issuesUrl": "https://gitlab.com/curben/phishing-filter/issues",
     "licenseId": 8,
     "name": "phishing-filter (Suricata)"
-  }
+  },
+
+  {
+
+"description": "Personal Filters to remove World Wide Web Annoyances for a better experience.",
+
+"homeUrl": "https://raw.githubusercontent.com/troysjanda/MyBlockLists/master/UBOXPI",
+
+"id": 2496,
+
+"licenseId": 4,
+
+"name": "XPI's Filter List",
+
+"submissionUrl": "https://github.com/troysjanda/MyBlockLists/issues"
+
+},
```

maybe need to remove this comma at the end. JSON is finnicky. 😀
```suggestion
}
```